lib/rack/tracker/google_adwords_conversion/google_adwords_conversion.rb

```diff
@@ -0,0 +1,15 @@
+class Rack::Tracker::GoogleAdwordsConversion < Rack::Tracker::Handler
+
+  class Conversion < OpenStruct
+  end
+
+  self.position = :body
+
+  def render
+    Tilt.new( File.join( File.dirname(__FILE__), 'template', 'google_adwords_conversion.erb') ).render(self)
+  end
+
+  def self.track(name, *event)
+    { name.to_s => [event.last.deep_stringify_keys.merge('class_name' => event.first.to_s.capitalize)] }
```

Comment:
the `deep_stringify_keys` is no longer necessary with the new centralized way of doing this